src/applications/vaos/containers/ExpressCareFormPage.jsx

```diff
@@ -1,69 +1,177 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
+import { scrollAndFocus } from '../utils/scrollAndFocus';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
-import { FETCH_STATUS, EXPRESS_CARE_ERROR_REASON } from '../utils/constants';
+import phoneUI from 'platform/forms-system/src/js/definitions/phone';
+import {
+  FETCH_STATUS,
+  EXPRESS_CARE_REASONS,
+  EXPRESS_CARE_ERROR_REASON,
+} from '../utils/constants';
 import FormButtons from '../components/FormButtons';
+import ExpressCareReasonField from '../components/ExpressCareReasonField';
 
 import * as actions from '../actions/expressCare';
 
+const pageKey = 'form';
+const pageTitle = 'Select a reason for your Express Care request';
+
+const initialSchema = {
+  type: 'object',
+  properties: {
+    'view:textObject': {
```

Comment:
I think you can remove these two `view:` fields now

Reply:
Removed